### PR TITLE
Change nonce types and examples to string

### PR DESCRIPTION
**Pull request by tommycodebox**
## 🧠  Description
- Change nonce used for auth to be of string type and update examples correspondingly

### CODE REVIEW

File: src/spec/swagger-v11--submitted.json [hunk 1]
```diff
@@ -2282,7 +2282,7 @@
         "nonce":{
           "type":"number",
```
Comment (Matas555) [inline src/spec/swagger-v11--submitted.json:2283]: Shouldn't this be a string too?

Comment (Neozaru) [reply src/spec/swagger-v11--submitted.json:2283]: That one might be a `nonce` related to withdrawals themselves and maybe not related to auth

Comment (tommycodebox) [reply src/spec/swagger-v11--submitted.json:2283]: I am not sure, this one is part of `withdrawalPayload` and I have seen another one in `starkOrder` payload, which is also number, but I only changed the ones that also have `signature` in body or query params, does that makes sense or that stark order nonce is also string?

Comment (Matas555) [reply src/spec/swagger-v11--submitted.json:2283]: Honestly don't know, but if the task is auth only, then the PR is good enough for me :D

**Review by Matas555 — APPROVED** [review]

**Pull request merged**